.github/workflows/CD.yml

```diff
@@ -0,0 +1,31 @@
+name: CD
+
+on:
+  release:
+    types: [released, prereleased]
+  workflow_dispatch:  # allows running workflow manually from the Actions tab
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        pip install -U pip setuptools wheel
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
@thomasjahoda, ref #3, Travis is giving up... For this PR, as I don't have access to repo settings, could you add your PyPI credentials [here](https://github.com/thomasjahoda/cookiecutter_project_upgrader/settings/secrets/actions)? Afterwards, when this file is present on `master` branch, we can trigger an upload of the new v1.3.0 manually from the `Actions` tab of the repo, or simply create a new GitHub release v1.3.1 which will auto-trigger the continuous deployment :) no more need to manually enter a new tag in the source, [setuptools_scm](https://github.com/pypa/setuptools_scm/) reads it automatically from the git tags :)